synchronizer/l1_workers_decorator_limit_retries_by_time.go

```diff
@@ -0,0 +1,70 @@
+package synchronizer
```

Comment:
I don't know where to put it but you can take the opportunity to rename the files removing the _ in the names (Snake Case)

Reply:
the files are snake_case